pkgs/development/ocaml-modules/nocrypto/default.nix

```diff
@@ -52,7 +52,7 @@ stdenv.mkDerivation rec {
   ];
 
   nativeBuildInputs = [ ocaml findlib ocamlbuild cc-wrapper ];
-  buildInputs = [ ocamlbuild findlib topkg cpuid ocb-stubblr ];
+  buildInputs = [ ocamlbuild findlib topkg cpuid ocb-stubblr ppx_deriving ];
```

Comment:
`ppx_deriving` should be propagated.

Reply:
This broke `jackline`, see: https://hydra.nixos.org/build/107730405

Will fix there: https://github.com/NixOS/nixpkgs/pull/74437